.github/workflows/diamond-storage-check.yml

```diff
@@ -0,0 +1,60 @@
+name: Check For Diamond Storage Changes
+
+on:
+  push:
+    branches:
+      - development
+    paths:
+      - '**.sol'
+  pull_request:
+    paths:
+      - '**.sol'
+
+jobs:
+  provide_contracts:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      - id: set-matrix
+        working-directory: packages/contracts
+        run: |
+          forge tree | grep -E '^src/dollar/libraries/Lib' | cut -d' ' -f1 | xargs basename -a | cut -d'.' -f1 | xargs -I{} echo src/dollar/libraries/{}.sol:{} >> contracts.txt
+          echo "matrix=$(cat contracts.txt | jq -R -s -c 'split("\n")[:-1]')" >> $GITHUB_OUTPUT
+
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+
+  check_storage_layout:
+    needs: provide_contracts
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        contract: ${{ fromJSON(needs.provide_contracts.outputs.matrix) }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
+        with:
+          version: nightly
+        
+      - name: Check For Diamond Storage Changes
+        uses: gitcoindev/foundry-storage-check@main
```

Comment:
```suggestion
        uses: ubiquity/foundry-storage-check@main
```

Reply:
just saw the commit from you, thank you!